docs/pages/link.js

```diff
@@ -624,20 +638,51 @@ However, Link's underline style can be overridden at any time using the \`underl
 `}
           />
         </MainSection.Subsection>
-        <MainSection.Subsection
-          title="Rel"
-          description={`\`rel\` is optional. Use "nofollow" for external links to specify to web crawlers not follow the link. Don't use "nofollow" with urls redirecting to any Pinterest site.`}
-        >
-          <MainSection.Card
-            defaultCode={`
+      </MainSection>
+
+      <MainSection.Subsection
+        title="externalLinkIcon and rel"
+        columns={2}
+        description={`An external link, also called an outbound link, is a link from Pinterest to a different website. External links require specific SEO, visual, and accessibility treatments.
+
+\`rel\` is optional. Use "nofollow" for external links to specify to web crawlers not follow the link. Don't use "nofollow" with urls redirecting to any Pinterest domain or subsite.
```

Comment:
nit: there's some inconsistency on this page between "subsite" and "sub-site". I don't have an opinion either way, just that we use the same term throughout

Reply:
DONE